lib/config/environment.js

```diff
@@ -66,6 +66,7 @@ module.exports = {
     consumerSecret: process.env.CMD_TWITTER_CONSUMERSECRET
   },
   github: {
+    enterpriseURL: process.env.ENTERPRISE_URL,
```

Comment:
We use `CMD` as environment variable prefix, so this should be `CMD_GITHUB_ENTERPRISE_URL`

Reply:
oops.... I was wrong. fixed.